TeamCode/src/main/java/org/firstinspires/ftc/teamcode/ManualDriveOpMode.java

```diff
@@ -14,17 +14,18 @@
  * becomes relative to the field as opposed to the robot. You can
  * reset the forward heading by pressing "x".
  */
-@TeleOp(name = "Mecanum")
+@TeleOp(name = "Manual Drive")
```

Comment:
Bad move, i think it should be 
```suggestion
@TeleOp(name = "Manual Drive 2")
```